contrib/sync_namespaces_jobs.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+import os
+
+import yaml
+
+DST_DIR = "/tmp/tron-servdir"
+#DST_DIR = "/nail/home/chl/tron-servdir"
+SRC_DIR = "/nail/etc/services/tron/prod"
+
+#SRC_DIR = "/nail/home/chl/git/yelpsoa-configs/tron/prod"
+#DST_DIR = "/nail/home/chl/Tron/contrib/tron-servdir"
+
+
+def main():
+    print(
+        "This script will load the same amount of namepance and jobs from tron-prod to tron-playground."
+    )
+    print("Please make sure you run it at playground")
+
+    for (dirpath, dirname, filenames) in os.walk(SRC_DIR):
+        for filename in filenames:
+            print('filename = {}'.format(filename))
+            filepath = os.path.join(SRC_DIR, filename)
+            if os.path.isfile(filepath) and filepath.endswith(".yaml"):
+                with open(filepath, "r") as f:
+                    config = yaml.load(f)
+                    if filename == "MASTER.yaml":
+                        for key, _ in config.items():
+                            if key != 'jobs':
+                                del config[key]
+
+                    jobs = config.get("jobs", [])
+                    if jobs is not None:
+                        for job in jobs:
+                            job['node'] = "localhost"
+                            if 'monitoring' in job:
+                                del job['monitoring']
+                            for action in job.get("actions", []):
+                                action['command'] = '/bin/true'
```

Comment:
Some sleeping might be more realistic, keeping track of running jobs probably adds some load.